Travel Scheduler/Helper Files/APIManager.m

```diff
@@ -286,6 +286,30 @@ - (void)getPhotoFromReference:(NSString *)reference withCompletion:(void (^)(NSU
     [task resume];
 }
 
+- (void)getWebsiteLinkOfPlaceWithId:(NSString *)placeId withCompletion:(void (^)(NSString *placeWebsiteString, NSError *error))completion
```

Comment:
I'm just curious, but this seems to be the same API call as the getCompleteInfoOfLocationWithName and that API call also gives the website. Is there a reason why you made a new one and didn't just reuse that one?

Reply:
Yeah. I also thought we had already gotten the website url from the getCompleteInfoOfLocationWithName API call.